# 리스트에서의 IndexError 예외
list_a = [273, 32, 103]
list_a[3] # IndexError: list index out of range
